### PR TITLE
Added Timer.running property

### PR DESCRIPTION
**Pull request by Rodsevich**
I had to use this and end up using an auxiliary bool variable, not so fancy :/

### CODE REVIEW

**Comment by Rodsevich** [comment]
You are right, excellent idea!
[Fixed mess](https://github.com/DirectMyFile/console.dart/pull/16/commits/10a601a5e4f07c0e1f3d18bfa2acd54c17bcee12)

File: lib/src/timer.dart [hunk 1]
```diff
@@ -6,9 +6,8 @@ class TimeDisplay {
   bool _isStart = true;
   String _lastMsg;
   Timer _updateTimer;
-  bool _running = false;
 
-  bool get running => _running;
+  bool get isRunning => (_watch?.isRunning == null) ? false : _watch.isRunning;
```
Comment (Pacane) [inline lib/src/timer.dart]: I think it can be simplified to : `watch == null ? false : watch.isRunning`

**Comment by Rodsevich** [comment]
You were right once again. Thanks!

**Comment by Pacane** [comment]
@Rodsevich After sleeping on this, I knew there was an alterate way to do this.

`bool get isRunning => _watch?.isRunning ?? false;`

**Comment by Rodsevich** [comment]
Would you believe I ended up on same? I guess you like code golfing a lot like me hugh? :rofl:

**Pull request merged**